tests/by-util/test_cp.rs

```diff
@@ -3701,3 +3703,57 @@ fn test_cp_no_such() {
         .fails()
         .stderr_is("cp: 'no-such/' is not a directory\n");
 }
+
+#[cfg(all(unix, not(target_os = "macos")))]
```

Comment:
I think it should be the same as for `test_acl_preserve`:
```suggestion
#[cfg(all(unix, not(any(target_os = "android", target_os = "macos"))))]
```